### ISSUE TITLE
changing value of C has no effect in svm_author_id exercise

### ISSUE THREAD

**Issue by chrisdraws** [udacity/ud120-projects#282]
I'm working on the svm_author_id exercise and I'm doing this -> 

`clf = SVC(C=1000.0,kernel="rbf" )`

the value of C is having no effect regardless of what I pass into it.  it doesn't change the accuracy at all.  What mistake am I making?

**Comment by digicom1978** [comment]
same thing happened to me. All values of C gave same accuracy.

**Comment by ViAsmit** [comment]
@chrisdraws 
Could you please provide the rest of the code. So that I can understand where the Problem is...

**Comment by aaakashkumar** [comment]
Could be strongly linked to #283 

The output is 0.8998862343572241 for all values of C.

**Comment by isislia** [comment]
I get the same issue, changing C does not seem to be changing the accuracy. This is the code I'm using 
`from sklearn import svm

features_train = features_train[:int(len(features_train)/100)]
labels_train = labels_train[:int(len(labels_train)/100)]

clf = svm.SVC(kernel = 'rbf', gamma = 'auto', C=10)

t0 = time()
clf.fit(features_train, labels_train) 
print("training time:", round(time()-t0, 3), "s")

t1 = time()
pred = clf.predict(features_test)
print("prediction time:", round(time()-t1, 3),"s")

from sklearn.metrics import accuracy_score 
acc = accuracy_score(labels_test, pred)
print(acc) `

**Comment by jayneel-shah18** [comment]
I too faced the same problem but I guess if we go by the docs, higher value of C should be more accurate and lower value of C though larger margin but will be less accurate.

**Comment by sihaog** [comment]
Same problem here. I couldn't get the "correct"   
Finally solved with adding `gamma='auto'` to the classifier.
`clf = SVC(kernel='rbf', C=c_val, gamma='auto')`

Docs show that in 0.22 the default for gamma changed from 'auto' to 'scale'. If you set gamma to auto, you get the lower accuracy expected by the mini quiz
https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html

_Originally posted by @paulgrainger85 in https://github.com/udacity/ud120-projects/issues/283#issuecomment-673138529_

**Comment by Tiamiyu1** [comment]
> Same problem here. I couldn't get the "correct" accuracy for the quiz. Finally solved with adding `gamma='auto'` to the classifier. `clf = SVC(kernel='rbf', C=c_val, gamma='auto')`
> 
> Docs show that in 0.22 the default for gamma changed from 'auto' to 'scale'. If you set gamma to auto, you get the lower accuracy expected by the mini quiz https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html
> 
> _Originally posted by @paulgrainger85 in [#283 (comment)](https://github.com/udacity/ud120-projects/issues/283#issuecomment-673138529)_

Worked!
